docs/bigquery/snippets.py

```diff
@@ -89,476 +97,556 @@ def do_something_with(_):
     # [END client_list_datasets]
 
 
-def test_dataset_create(client, to_delete):
+def test_create_dataset(client, to_delete):
     """Create a dataset."""
-    DATASET_NAME = 'dataset_create_%d' % (_millis(),)
-
-    # [START dataset_create]
-    dataset = client.dataset(DATASET_NAME)
-    dataset.create()              # API request
-    # [END dataset_create]
-
-    to_delete.append(dataset)
+    DATASET_ID = 'create_dataset_%d' % (_millis(),)
 
+    # [START create_dataset]
+    dataset_ref = client.dataset(DATASET_ID)
```

Comment:
We might want to put a comment before `dataset_ref` line and after `START` line like

    # DATASET_ID = 'dataset_ids_are_strings'

Reply:
Added in e30c19a